Gulpfile.js

```diff
@@ -142,7 +142,20 @@ const CLIENT_TESTS_SAUCELABS_SETTINGS = {
 
 const CLIENT_TEST_LOCAL_BROWSERS_ALIASES = ['ie', 'edge', 'chrome', 'firefox', 'safari'];
 
-const PUBLISH_TAG = JSON.parse(fs.readFileSync(path.join(__dirname, '.publishrc')).toString()).publishTag;
+const PUBLISH_VERSION_RE = /^\d+\.\d+\.\d+(-alpha\.\d+)?$/;
+const PUBLISH_TAGS       = getPublishTags();
+const PUBLISH_REPO       = 'testcafe/testcafe';
+
+function getPublishTags () {
```

Comment:
Minor structural remark.
You call `getPublishTags` function before its declaration. 
I propose to change function signature as `function getPublishTags (packageInfo) {...}` and move this function to a separate file `docker/get-publish-tags`.